src/test/java/com/lambdaschool/foundation/SeedDataTest.java

```diff
@@ -0,0 +1,148 @@
+package com.lambdaschool.foundation;
```

Comment:
You should update this test SeedData file to the one I have in main. Since we are not actually interacting with the database, all the data we need for the test should be mocked up in the test file.

Reply:
I have corrected this issue.